scripts/deploy_bug.py

```diff
@@ -41,9 +42,14 @@ def fetch_history_from_github(user, repo, from_sha):
     return fetch(GITHUB_API + 'repos/%s/%s/compare/%s...master' % (user, repo, from_sha))
 
 
-def print_row(*args):
-    row_template = '%-13s ' * len(args)
-    print(row_template % args)
+def fetch_current_revision(host):
+    # Try /__version__ and fall back to /api/Status/
+    resp = fetch(host + '/__version__')
+    if 'commit' in resp:
+        return resp['commit']
+
+    resp = fetch(host + '/api/Status/')
+    return resp['socorro_revision']
```

Comment:
Socorro's current infrastructure uses `/api/Status/`, but the new infrastructure uses `/__version__`. I figured I'd add support for both so this is more future proof.